lib/web_ui/test/engine/pointer_binding_test.dart

```diff
@@ -18,22 +18,43 @@ List<ui.PointerData> _allPointerData(List<ui.PointerDataPacket> packets) {
   return packets.expand((ui.PointerDataPacket packet) => packet.data).toList();
 }
 
+/// Helper class for encapsulating test skip information with test `context`.
+///
+/// Some tests should be skipped depending on the browser and the OS. This class
+/// is used for combining [shouldSkip] information with the `context`.
+/// See: [_PointerEventContext] [_MouseEventContext] [_TouchEventContext].
+/// TODO: https://github.com/flutter/flutter/issues/60033
+class _BrowserUnitTestContext<T extends _BasicEventContext> {
```

Comment:
I also considered using Tuple: https://pub.dev/packages/tuple

@mdebbar let me know if this looks good to you or should I changed to Tuple2.

Reply:
I think this is way more readable than Tuple2, because you can name and document the fields.